integration-tests/src/test/java/tachyon/worker/DataServerIntegrationTest.java

```diff
@@ -314,13 +349,15 @@ private DataServerMessage request(final ClientBlockInfo block, final long offset
     }
   }
 
-  @Test
-  public void tooLargeOffset() throws IOException {
-    final int length = 10;
-    int fileId =
-        TachyonFSTestUtils.createByteFile(mTFS, "/readTooLarge", WriteType.MUST_CACHE, length);
-    ClientBlockInfo block = mTFS.getFileBlocks(fileId).get(0);
-    DataServerMessage recvMsg = request(block, length * 2, 1);
-    assertError(recvMsg, block.blockId);
+  /**
+   * returns the MasterBlockInfo of the first block of the file
```

Comment:
s/returns the MasterBlockInfo of the first block of the file/Returns the MasterBlockInfo of the first block of the file./